src/components/UniversalNavbarExpanded/SampleContent.js

```diff
@@ -1,13 +1,17 @@
-import uuidv4 from 'uuid/v4'
-export const LINKS = {
+/**
+ * Sample of UniversalNavbarExpanded links.
+ * For demonstration in styleguide & test files (.md/.test.js files) only!
+ * Real implementation of this structure should use uuid() for id values!
+ */
+export const CMS_LINKS = {
   // These are used e.g. in the logo, icons and bottom of mobile nav:
   INDEX: { href: '/' },
-  TERM: {
+  CTA: {
     href: '/term',
     title: 'Check my price',
   },
   // TALK: {
-  //     id: uuidv4(),
+  //     id: 'NAVLINKS_MOCK_ID_',
   //     href: '/',
   //     title: 'Talk to us',
   //   },
```

Comment:
Can this be removed? It's commented out

Reply:
Yeah, it'll be added back later but I'll remove until it is.

Reply:
8b2c0d1